src/js/tracks/text-track-settings-controls.js

```diff
@@ -28,14 +26,15 @@ class TrackSettingsControls extends Component {
 
     this.addChild(resetButton);
 
+    const doneText = this.localize('Done');
     const doneButton = new Button(player, {
-      controlText: defaultsDescription,
+      controlText: doneText,
```

Comment:
if it's repeating the text content, we shouldn't need control text (though, worth verifying with a screen reader).

Reply:
I've tested these changes in NVDA & Voiceover, there's no repeat, the changes here just ensure the done and reset buttons have the correct text. Previously the done button had the same text as the reset button, which was misleading.